source/builders/filters.txt

```diff
@@ -282,7 +282,7 @@ polygon:
                                       Position(0, 0)))
    geoWithin("geo", polygon)
 
-Similarly, this example creates a filter that selects all documents
+Similarly, The following example creates a filter that selects all documents
```

Comment:
Suggestion:

Aside from the capitalized "T" in "the", I think omitting "Similarly" could be more effective as what the similarity is could be misinterpreted.

Reply:
Oops, find and replace error. Will just remove the similarly